src/test/java/uk/gov/hmcts/reform/sscs/service/VenueDataLoaderTest.java

```diff
@@ -122,4 +125,26 @@ public void shouldGetEpimsIdForGivenPostcode() {
         String result = venueDataLoader.getActiveVenueDetailsMapByPostcode().get("MK9 2AJ").getEpimsId();
         assertEquals("815997", result);
     }
+
+    @Test
+    public void testActiveVenueEpimsIdsMapByRpcReturnsVenues() {
+        List<String> result = venueDataLoader.getActiveVenueEpimsIdsMapByRpc().get("SSCS Leeds");
+
+        assertFalse(result.isEmpty());
+        assertEquals(20, result.size());
+        assertEquals(Arrays.asList("517400", "449358", "563156",
```

Comment:
I know the class is using the actual venue data loader but I don't think we should be doing this. It would be better to use a mock loader so this test won't fail should any Leeds data change